client/src/pages/_document.tsx

```diff
@@ -1,26 +1,13 @@
-import { ServerStyleSheets } from '@material-ui/core/styles';
 import Document, { Head, Html, Main, NextScript } from 'next/document';
 import * as React from 'react';
 
 export default class MyDocument extends Document {
   static async getInitialProps(ctx: any) {
```

Comment:
Remove whole `getInitialProps` function here

Reply:
Sure , @Zeko369

Reply:
@Zeko369 I've removed the getInitalProps function completely from _document.tsx. Now it only has the render function.